examples/change_baudrate.py

```diff
@@ -0,0 +1,85 @@
+from typing import Optional
+
+from ingenialink import CAN_BAUDRATE, CAN_DEVICE
+
+from ingeniamotion.exceptions import IMException
+from ingeniamotion.motion_controller import MotionController
+
+
+def establish_canopen_communication(
+    mc: MotionController,
+    device: CAN_DEVICE,
+    channel: int,
+    baudrate: CAN_BAUDRATE,
+    dictionary_path: str,
+    node_id: Optional[int],
+):
+    print("Finding the available nodes...")
+    node_id_list = mc.communication.scan_servos_canopen(
+        device,
+        baudrate=baudrate,
+        channel=channel,
+    )
+    if not node_id_list:
+        return False
+
+    print(f"Found nodes: {node_id_list}")
+    if node_id is None:
+        print("Node ID is selected automatically.")
+        node_to_connect = node_id_list[0]
+    else:
+        print("Node ID is selected manually.")
+        node_to_connect = node_id
+
+    print("Starts to establish a communication.")
+    mc.communication.connect_servo_canopen(
+        device,
+        dictionary_path,
+        node_to_connect,
+        baudrate=baudrate,
+        channel=channel,
+    )
+    print(f"Drive is connected with {baudrate} baudrate.")
+    return True
```

Comment:
Add a docstring to explain what the functions do and what the returned value is. As we said in old PRs, remember it is an example, not a script, comments could be useful than prints.